source/codegen/service_helpers.py

```diff
@@ -245,6 +245,8 @@ def _create_param(parameter, expand_varargs=True, repeated_parameters=None):
         else:
             return "..."
     elif common_helpers.is_array(type):
+        if type == "void *":
```

Comment:
Make sure this works OK with the other pending changes to this code.

Reply:
I copied this part from their pending changes and updated the metadata to work with it so it should be good to go.